spec/requests/users_controller_spec.rb

```diff
@@ -147,6 +147,23 @@
         expect(response).to redirect_to(destination_url)
       end
     end
+
+    context 'when cookies does not contain a destination URL but users was invited to topic' do
+      let(:invite) { Fabricate(:invite) }
+      let(:topic) { Fabricate(:topic) }
+
+      before do
+        TopicInvite.create!(topic: topic, invite: invite)
+        Fabricate(:invited_user, invite: invite, user: email_token.user)
+        invite.reload
```

Comment:
Just curious, why do we need to reload the invite here?

Reply:
I think it was necessary to ensure that `invited_user` field was present.